scripts/build/docker/Dockerfile.template

```diff
@@ -31,24 +31,32 @@ RUN apt-get update \
     python-software-properties \
     unzip \
     xvfb \
-
-    zip   \
+    zip \
     rpm
 
 # Install a C++11 compiler
 RUN add-apt-repository ppa:ubuntu-toolchain-r/test \
   && apt-get update && apt-get install -y gcc-4.8 g++-4.8 \
   && update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-4.8 50
 
-# NodeJS
-RUN curl -sL https://deb.nodesource.com/setup_6.x | bash - \
-  && apt-get install -y nodejs
+# Install Node@8.x.x
+RUN git clone https://github.com/tj/n && \
+  cd n && git checkout v2.1.4 && make install && \
+  cd .. && n 8.1.0
```

Comment:
It's real speedy. I'd even argue that it's the fastest way to auto-install node. Also a most splendid node version manager.